docs/sources/tempo/traceql/trace-structure.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Trace structure and TraceQL
+menuTitle: Trace structure and TraceQL
+description: Learn about trace structure and TraceQL queries.
+weight: 200
+keywords:
+  - syntax
+  - TraceQL
+---
+
+# Trace structure and TraceQL
+
+The TraceQL language is designed to use the same syntax and semantics as PromQL and LogQL, where possible.
+The differences being inherent to the very nature of searching spans and traces.
+
+## Trace structure
+
+Traces are telemetry data structured as trees.
+Traces are made of spans (for example, a span tree); there is a root span that can have zero to multiple branches that are called child spans.
+Each child span can itself be a parent span of one or multiple child spans, and so on so forth.
+
+![Trace_and_spans_in_tree_structure](/media/docs/tempo/traceql/trace-tree-structures-and-spans.png)
+
+In the specific context of TraceQL, a span has the following associated fields:
+
+- **name**: the span name
+- **duration**: difference between the end time and start time of the span
+- **status**: enum: `{ok, error, unset}`. For details, refer to [OTel span status](https://opentelemetry.io/docs/concepts/signals/traces/#span-status) documentation.
+- **kind**: enum: `{server, client, producer, consumer, internal, unspecified}`. For more details, refer to [OTel span kind ](https://opentelemetry.io/docs/concepts/signals/traces/#span-kind) documentation.
+- Attributes
+
+The first four properties are *intrinsics*.
+They are fundamental to the span.
+
+*Attributes* are custom span metadata in the form of key-value pairs.
+There are two types of attributes: **span attributes** and **resource attributes**.
+
+*Span attributes* are key-value pairs that contain metadata that you can use to annotate a span to carry information about the operation it's tracking.
+For example, in an eCommerce application, if a span tracks an operation that adds an item to a user’s shopping cart, the user ID, added item ID and cart ID can be captured and attached to the span as span attributes.
+
+A *Resource attribute* represents information about an entity producing the span.
+For example, a span created by a process running in a container deployed by Kubernetes could link a resource that specifies the cluster name, namespace, pod, and container name.
```

Comment:
The SDK and version might also be good candidates for examples.

Reply:
Do you think this is something we should add in this PR or open another doc issue to add those examples? I'd like to get this content in, if we can.

Reply:
Let's take what you've got and we can sync if we need to.

Reply:
I've added it as a line item to my TraceQL doc epic.